libs/core/src/lib/select/select.interface.ts

```diff
@@ -1,8 +1,12 @@
-import { EventEmitter, InjectionToken } from '@angular/core';
+import { EventEmitter, InjectionToken, QueryList, ElementRef } from '@angular/core';
+
 import { OptionComponent } from './option/option.component';
-import { OptionStatusChange } from './select.component';
+import { FdSelectChange } from './select.component';
 import { DialogConfig } from '../dialog/utils/dialog-config.class';
 import { MobileMode } from '../utils/interfaces/mobile-control.interface';
+import { MobileModeConfig } from '../utils/public_api';
+import { OptionsInterface } from './options.interface';
+import { LiveAnnouncer } from '@angular/cdk/a11y';
 
 export const SELECT_COMPONENT = new InjectionToken<string[]>('SelectInterface');
```

Comment:
Why `string[]` and not `new InjectionToken<SelectInterface>('SelectInterface')`?